src/java/org/apache/cassandra/db/streaming/CassandraCompressedStreamReader.java

```diff
@@ -73,7 +73,7 @@ public SSTableMultiWriter read(DataInputPlus inputPlus) throws Throwable
         SSTableMultiWriter writer = null;
         try (CompressedInputStream cis = new CompressedInputStream(inputPlus, compressionInfo, ChecksumType.CRC32, cfs::getCrcCheckChance))
         {
-            TrackedDataInputPlus in = new TrackedDataInputPlus(cis);
+            RateLimitedTrackedDataInputPlus in = new RateLimitedTrackedDataInputPlus(cis, -1, limiter, totalSize, 1 << 16);
```

Comment:
the 64kb is kinda hard coded in all places using this right? maybe make it a constant and overridable with yaml or system property